.babelrc.js

```diff
@@ -1,22 +1,41 @@
 const r = m => require.resolve(m)
 
-function preset(_, options = {}) {
-  const PRODUCTION = process.env.NODE_ENV === "production"
+function preset(context, options = {}) {
+  const { browser = false, debug = false } = options
+  const { NODE_ENV, BABEL_ENV } = process.env;
+
+  const PRODUCTION = (BABEL_ENV || NODE_ENV) === "production"
+
+  const browserConfig = {
+    useBuiltIns: false,
+    targets: {
+      browsers: PRODUCTION
+        ? [`last 4 versions`, `safari >= 7`, 'ie >= 9']
+        : [`last 2 versions`, `not ie <= 11`, `not android 4.4.3`],
```

Comment:
This can be whatever, but figured it wasn't worth compiling to ie9 if it was dev